postgis/postgis.sql.in

```diff
@@ -3678,6 +3678,12 @@ CREATE OR REPLACE FUNCTION pgis_geometry_accum_transfn(pgis_abs, geometry, float
 	AS 'MODULE_PATHNAME'
 	LANGUAGE 'c';
 
```

Comment:
I wonder if it's possible to merge these using default values of NULL on the spare parameters.